c_emulator/jtag_dtm.cpp

```diff
@@ -0,0 +1,276 @@
+#include <stdio.h>
+#include "jtag_dtm.h"
+#include "remote_bitbang.h"
+#include "riscv_sail.h"
+#include "sail.h"
+
+#if 0
+#define D(x) x
```

Comment:
Can you give this a longer name? You could also do

```
#ifdef JTAG_LOGGING
#define jtag_log(...) fprintf(stderr, __VA_ARGS__)
#else
#define jtag_log(...)
#endif
```

Or something like that.